django_opentracing/tracer.py

```diff
@@ -1,12 +1,26 @@
 from django.conf import settings
 import opentracing
 
+django_tracer = None
+
+def get_tracer():
```

Comment:
I don't think there's need for this function, because of it always returning `opentracing.tracer` - moreover, I don't think we *need* to expose it in the public api. Or is there any use scenario where you need it?